### DISCUSSION TITLE
prices/sum of all parts of a project in BOM list...?

### DISCUSSION THREAD

**Discussion by lembi0** [Part-DB/Part-DB-server#576] [Q&A]
I just installed the latest version (1.11.3) of PartDB (upgraded from 0.6). 
Version 0.6  showed the prices and sum of all parts of a project - 1.11 doesn't? There are a lot of options (see below) to configure all the rows which should be shown in the table, but no "price" option to choose?

Is it no longer intended to show a 'total' price there (or anywhere else) or am i just unable to find it :)

Alex

![shareX_328575124](https://github.com/Part-DB/Part-DB-server/assets/6428678/a6f44ba4-da46-4599-bd47-7dfc95cf9917)

**Comment by ManuelFollmann** [comment]
That was a usefull feature and I would appreciate to have it again.

**Comment by MelvinValerius** [comment]
That is the very reason for which I wanted to try the program and now - after I added all the parts and created the projects - I find out that feature does not exist anymore.
Will it be (Re)added because otherwise I sadly wasted a huge amount of time?

**Comment by AndreasWickstrom** [comment]
I have just started to work with Part-db and i can just agree that it would be nice to have that price feature.
